_posts/rosi_bajari/2018-07-13-genomics.md

```diff
@@ -0,0 +1,102 @@
+
+
+## So, uh, what did that last thing actually mean!?
+
+  
+NGS is probably what people are referring to when they talk about “genomics”. Massively parallel, high-throughput Next Generation Sequencing is sequencing on steroids - It's faster than anything else, cooler than anything else, and produces a *tonne* of data. The basic idea of NGS this: Get someone's DNA, shred it into a lot of pieces, make a lot of copies of it, and then sequence it with a machine. Sounds easy right?
+
+You can think of DNA sequencing like this:
+
+>You need to find a single paragraph in a book. To find the paragraph you are interested in, you first have to shred all its pages. Now you have to glue it back together, but a lot of the paragraphs look really similar since they have words like “the”, “when”, and "too" everywhere. Also, maybe you accidently got some of another books pages mixed in, so you need to clean those out while you sort and glue things together.
+
+  
+Hmm, not so easy. How exactly does one accomplish such a monumental task then? With this simple recipe:
+
+1.  Use super cool science to harvest some DNA.
+    
+2.  Since DNA is so small, we like to increase our chances of success of sequencing it by amplifying it into many copies.
+    
+3.  Then, it goes into a super fancy sequencing machine that probably operates on a sequencing-by-synthesis methods. Individual base pairs (A-T-C-G) are attached to a special fluorescent dye that a camera reads as each base pair gets individually added to a DNA strand. Remember that amplification in step 2? This is why - the camera might not be able to read the fluorescent signal from one base, but it can with many signal powers combined. At the end, the sequencing machine spits out FASTQ data files that contains “unaligned reads” - small text strings of base pairs.
+    
+4.  So now you have a (really) giant pile of small strings of DNA that you need to “align” or glue back together into the original DNA strand. People have developed... [a few algoritms](https://en.wikipedia.org/wiki/List_of_sequence_alignment_software) for this. Across the board, there are basic inputs: the FASTA files you just made, and a reference genome. The reference genome is a widely accepted standard template that says _exactly_ where each gene is located. Reads are compared to the reference genome as a template to reconstruct the original genome and output as a new file - a BAM file of “aligned reads”.
+
+Congratulations - Your genome is now sequenced! So what now?
+
+<center>
+  <figure >
+      <img src="{{site.urlimg}}rosi_bajari/hitchhikers/basic-image2.png"/>
+  </figure>
+</center>
+
+## Spot the Difference!
+
+There’s all kinds of fun stuff you can do! Usually, the first thing that researchers do is try and look for any differences in the genome, called mutations, through a methods like “variant calling”.
```

Comment:
It needs to provide a bit more context here. `Mutation` is usually (although not always) used for changes that are harmful or in the context of cancer genomics, it's more of referring to somatic changes.

`look for any differences in the genome` needs to be more clear about `differences` between what and what. For examples, `differences` among individuals in common populations are usually referred as `polymorphism` (or `variation`; `differences` between mutant allele and a wide type allele are referred as `mutations` etc.

You may also want to specifically mention that in cancer genomics sequence changes between normal cell and tumor cell are referred as `somatic mutations`.